tests/python_api/audio_unit_test.py

```diff
@@ -116,8 +155,9 @@ def main():
     if not rocal_cpu:
         print("The GPU support for Audio is not given yet. Running on CPU")
         rocal_cpu = True
-    if audio_path == "":
-        audio_path = f'{rocal_data_path}/rocal_data/audio/wav/'
+    if audio_path == "" and file_list == "":
```

Comment:
empty() is better here
```suggestion
    if audio_path == "" and file_list == "":
```

Reply:
Changed to not